envs/environment.yaml

```diff
@@ -78,6 +78,7 @@ dependencies:
   # - git+https://github.com/pypsa/pypsa.git#egg=pypsa
   - git+https://github.com/davide-f/powerplantmatching.git@new_pypsa_africa#egg=powerplantmatching
   - git+https://github.com/davide-f/atlite.git@master
+  - git+https://github.com/ekatef/PyPSA.git@master
```

Comment:
Do we need this?

Reply:
Unfortunately, yes. We need a custom linetype for HVDC lines. The only proper way to do this seems to be a PR to the main PyPSA repository. But I have first to find the reference data on electrical properties of the HVDC cables which wasn't done yet

Reply:
Thank you :) Ok!